src/perf/lib/RpsClient.cpp

```diff
@@ -114,13 +114,13 @@ RpsClient::Start(
                     Event);
         };
 
-    Connections = UniquePtr<HQUIC[]>(new(std::nothrow) HQUIC[ConnectionCount]);
+    Connections = UniquePtr<ConnectionScope[]>(new(std::nothrow) ConnectionScope[ConnectionCount]);
     if (!Connections.get()) {
         return QUIC_STATUS_OUT_OF_MEMORY;
     }
 
     for (uint32_t i = 0; i < ConnectionCount; i++) {
-        Connections[i] = nullptr;
+        Connections[i].Handle = nullptr;
```

Comment:
This for loop is unnecessary, no? It should call the constructor, which will null out the handle pointers.

Reply:
In theory yes, but I'm not sure I 100% trust that kernel mode will do that. I can double check.